src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -851,6 +925,23 @@ public void SarifTransformerTests_ToCurrent_NotificationExceptionWithStack()
           ]
         }
       ],
+      ""logicalLocations"": {
+        ""Rules.SecureHashAlgorithmRule.Evaluate"": {
+          ""name"": ""Evaluate"",
+          ""kind"": ""some kind""
+        },
+        ""Rules.SecureHashAlgorithmRule.Register"": {
+          ""name"": ""Register""
+        },
+        ""ExecutionEngine.Engine.FooBar"": {
+          ""name"": ""EvaluateRule"",
+          ""fullyQualifiedName"": ""ExecutionEngine.Engine.EvaluateRule"",
+          ""kind"": ""another kind""
+        },
```

Comment:
Bug: These `name` and `fullyQualifiedName` values belong in the entry whose key is `ExecutionEngine.Engine.EvaluateRule"`.

Reply:
... and the `name` for this entry should be `"FooBar"`.

Reply:
This is for the stackFrame with the logicalLocationKey property set?

Reply:
If this is wrong, can you check the last v1 stackFrame and tell me if it's valid

Reply:
Ok. That last stack frame, with
```
"fullyQualifiedLogicalName": "ExecutionEngine.Engine.EvaluateRule"
"logicalLocationKey": "ExecutionEngine.Engine.FooBar"
```
is valid. I didn't notice where this `logicalLocation` came from.

I suggest you change `"FooBar"` to `"Evaluate-1"`. Because of the unrelated name, I didn't recognize that this was the "disambiguation" part of the test.

Reply:
So it's not meant to be the disambiguation test, it's meant to test the presence of logicalLocationKey. (I need to add a disambig. case)